custom_components/adaptive_lighting/const.py

```diff
@@ -201,6 +206,19 @@
     'documented defaults), or "configuration" (reverts to switch config defaults). ⚙️'
 )
 
+CONF_WHICH_SWITCH, DEFAULT_WHICH_SWITCH = "switch_type", "main"
```

Comment:
I imagine that "main" is not available in 99% of the installations. Should we have no default?

Reply:
that sounds like a good idea